tests/integration/test_install_basic.py

```diff
@@ -119,16 +119,16 @@ def test_install_without_dev(PipenvInstance, pypi):
 six = "*"
 
 [dev-packages]
-pytz = "*"
+tablib = "*"
```

Comment:
This replacing of `pytz` with `tablib` is a slightly different from the others, which are replacing `requests`.

pytz is not a dependency of pipenv, however it is a dependency of the test runner, so it is also added in `PYTHONPATH` in order to get the builds working in rpmbuild.

I can workaround this by creating a special venv with only the real dependencies of pipenv, and using that in `PYTHONPATH`.  But that is quite a bit more effort, and I doubt many distro maintainers would accept a much more complicated testing rig just so this one test passes - they would likely prefer to just mark this test as a skip.

Note I do not expect pipenv devs to always be avoiding requests and other dependencies in these tests.  It would be nice if they were avoided, especially requests, but it is not possible to predict in advance all of the dependencies, especially all of the test rig dependencies. 

I think the responsibility of creating fixup patches like this one falls on the de-vendor-ers after each release.

Also we should also be working on creating a job which blocks internet access so that the 'internet' marker becomes 100% reliable here -- so that de-vendor-ers dont feel that some of the failures they see might be due to 'no internet', and hopefully they investigate any failures more before shipping broken package updates to users.